sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/expressions.scala

```diff
@@ -138,56 +136,53 @@ object ConstantPropagation extends Rule[LogicalPlan] {
    *                    case of `WHERE e`, null result of expression `e` means the same as if it
    *                    resulted false
    * @return A tuple including:
-   *         1. Option[Expression]: optional changed condition after traversal
+   *         1. Expression: optional changed condition after traversal
```

Comment:
Remove the `optional`?

Reply:
Thanks, I forgot to update that. Fixed in https://github.com/apache/spark/pull/40268/commits/02440345fa88aac88d157f5d7660462d8e645918